src/app/(site)/movies/[movieId]/page.tsx

```diff
@@ -1,34 +1,64 @@
-import React from "react";
-import { notFound } from "next/navigation";
+'use client';
+// @refresh reset
+import React from 'react';
+import { notFound } from 'next/navigation';
+import Link from 'next/link';
 
-async function getMovie(id: any) {
-  const res = await fetch(`https://plankton-app-xhkom.ondigitalocean.app/api/movies/${id}`);
+async function getMovie(id: string) {
+  const res = await fetch(`http://localhost:3000/api/movies/${id}`, {
+    next: { revalidate: 1 },
+  });
+  return res.json();
+}
+async function getScreenings(id: string) {
+  const res = await fetch(`http://localhost:3000/api/movies/${id}/screenings`, {
+    next: { revalidate: 1 },
+  });
   return res.json();
 }
 
-export default async function movieInfo({ params: { movieId } }) {
+export default async function movieInfo({ params: { movieId } }: any) {
```

Comment:
You are using any as type in a couple of places in the code but I just point it out here. It is good practice to avoid any so typescript can do it's job. If possible change any to the right type.